server/tinode.conf

```diff
@@ -190,7 +190,14 @@
 				// See https://github.com/go-sql-driver/mysql#dsn-data-source-name for syntax.
 				"dsn": "root@tcp(localhost)/tinode?parseTime=true&collation=utf8mb4_unicode_ci",
 				// Name of the main database.
-				"database": "tinode"
+				"database": "tinode",
+				// Maximum number of open connections to the database. Default: 0 (unlimited).
+				// "max_open_conns": 64,
```

Comment:
Maybe not comment them out? There are reasonable values.

Reply:
SG. Done.